app-media-recorder.html

```diff
@@ -42,12 +42,13 @@
 <dom-module id="app-media-recorder">
   <script>
     (function() {
+      var supported = true;
```

Comment:
See above, a more descriptive name would probably be useful for folks wondering what this means when reading the method implementation.